### PR TITLE
Pd 757 fix wallet filter in transactions

### PR DESCRIPTION
**Pull request by danoctavian**
Fix transactions 'wallet' filter which was being ignored + do a removal of the migration filtering logic.

### CODE REVIEW

File: src/database/index.ts [hunk 1]
```diff
@@ -27,10 +27,8 @@ async function getTransactions(limit: number, offset: number, symbol: string,
   }
 
   const transactions = await qbDB.models.transaction.findAll({
-    where: {
-      txType: {$ne: 'migration'}
-
-    },
+    where: txFilters
+    ,
```
Comment (aaroncolaco) [inline src/database/index.ts]: comma on wrong line?

File: __tests__/integration/transactions.test.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ import utils from '../../src/lib/utils'
 import log from '../../src/logging'
 import APITesting from '../apiTesting'
 import TestPrivateChain from './testPrivateChain'
+import Test = jest.Test
```
Comment (aaroncolaco) [inline __tests__/integration/transactions.test.ts]: What does this line do?

File: __tests__/integration/transactions.test.ts [hunk 3]
```diff
@@ -522,31 +542,6 @@ describe('Transactions API Integration', () => {
     expect(transactionsAfterResponse.body.message).toContain('offset')
   })
 
-  it ('Filters out migration type of transaction from history and transactions list', async () => {
```
Comment (danoctavian) [inline __tests__/integration/transactions.test.ts]: removing migration type filtering logic and the respective test. as we've seen after migration, we don't actually record those transactions yet so best to simplify here

File: __tests__/integration/transactions.test.ts [hunk 4]
```diff
@@ -339,13 +339,33 @@ describe('Transactions API Integration', () => {
...
+    }
+
+    const sendTransactionResponse = await sendTransaction(rawTransactionParams)
+    await markTransactionAsMined(sendTransactionResponse.body.hash)
+
+    const transactionsHistoryByWallet =
+      await request(app).get(`/transactions?wallet=${ACCOUNTS[3].address}`)
+    const historicalTransactionsBySymbol = transactionsHistoryByWallet.body
+    expect(transactionsHistoryByWallet.status).toBe(HttpStatus.OK)
+    expect(historicalTransactionsBySymbol).toHaveLength(1)
```
Comment (danoctavian) [inline __tests__/integration/transactions.test.ts]: i added this more specific test that focuses on 1 wallet, the reason the test didn't catch the bug was that all transactions were executed by 1 wallet so adding the filter wouldn't reduce the list anyway. this proves the filter for a wallet that has only 1 tx.

**Pull request merged**